overview/default.nix

```diff
@@ -7,28 +8,32 @@
   inherit
     (builtins)
     any
+    attrNames
     attrValues
     concatStringsSep
     filter
     isList
-    mapAttrs
-    match
     readFile
+    stringLength
     substring
     toJSON
     toString
     ;
 
   inherit
-    (pkgs.lib)
+    (lib)
     concatLines
+    flattenAttrsDot
+    flip
     mapAttrsToList
     optionalString
-    unique
     ;
 
   empty = xs: assert isList xs; xs == [];
   heading = i: text: "<h${toString i}>${text}</h${toString i}>";
+  isPrefix = prefix: candidate: prefix == substring 0 (stringLength prefix) candidate;
+  concatStringsDot = concatStringsSep ".";
+  dottedLoc = option: concatStringsDot option.loc;
```

Comment:
I think this can go into a `let` where it's used. Doesn't feel like it's worth a lib function.

Reply:
Sure, will do.

Reply:
`dottedLoc` is used in two places, and that's already the smallest scope that covers both.